hyperspy/tests/component/test_components.py

```diff
@@ -175,6 +184,15 @@ def test_3d_signal(self):
         np.testing.assert_allclose(p.coefficients.map['values'],
                                    np.tile([0.5, 2, 3], (2, 5, 1)))
 
+    # For https://github.com/hyperspy/hyperspy/pull/1989
```

Comment:
What's the point of this commented out test?

Reply:
This test is broken at the moment if I remember correctly and this is a bit annoying to fix it with the current implementation of the polynomial component. I have been working on finishing #1989 so I would suggest to leave it as it is at the moment and uncomment this test in #1989.